napari/plugins/io.py

```diff
@@ -304,14 +316,22 @@ def _write_multiple_layers_with_plugins(
 
     Returns
     -------
-    list of str
-        A list of filenames, if any, that were written.
+    (written paths, writer name) as Tuple[List[str],str]
+
+    written paths: List[str]
+        Empty list when no plugin was found, otherwise a list of file paths,
+        if any, that were written.
+    writer name: str
+        Name of the plugin selected to write the data.
     """
 
     # Try to use NPE2 first
-    written_paths = _npe2.write_layers(path, layers, plugin_name, _writer)
-    if written_paths:
-        return written_paths
+    written_paths, writer_name = _npe2.write_layers(
+        path, layers, plugin_name, _writer
+    )
+    if written_paths or writer_name:
+        print('XXXXXXXXXX', (written_paths, writer_name))
```

Comment:
```suggestion
```

Assuming this was a debug statement. Alternatively use `logger.debug` if you think this will be useful in the future.

Reply:
I put the big X's there just so I wouldn't forget 🙄